.buildkite/pipeline-for-tags.yml

```diff
@@ -5,7 +5,7 @@ steps:
  - label: Check for new Tezos release
    if: build.source == "schedule" && build.branch == "master"
    commands:
-   - nix-shell ./scripts/shell.nix --run ./scripts/update-tezos.sh
+   - nix-unstable develop .#autorelease -c ./scripts/update-tezos.sh
```

Comment:
I think it's better to use the usual `nix` and `USE_NEWER_NIX=1` env variable so that we'll have to only remove this env variable once `nix` is updated build servers.